examples/webclient/blocking/src/main/resources/full-webclient-config.yaml

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
I think that this file is not exactly needed here. It was present in the original example to give overview of possible configuration settings, but this would be duplicity of it without any reason since blocking WebClient does not have any extra config options.

Reply:
fixed